src/JBTokens.sol

```diff
@@ -351,9 +350,6 @@ contract JBTokens is JBControlled, IJBTokens {
         override
         onlyControllerOf(projectId)
     {
-        // Can't transfer to the zero address.
-        if (recipient == address(0)) revert JBTokens_RecipientZeroAddress();
```

Comment:
Hmm transferring to `0` should really happen, because that should be `burnFrom(..)`

Reply:
it should really happen in any part of the protocol. `beneficiary` of `pay`, etc. do we wanna check everywhere? plus there are a bunch of non 0 addresses that are still bad. 

the proposal to remove the line is just consistency. if you fuck up you fuck up.